modules/handlers/callback_handlers.py

```diff
@@ -195,7 +210,8 @@ def approve_no_callback(info: dict, arg: None) -> Tuple[str, InlineKeyboardMarku
             logger.warning("Notifying the user on approve_no: %s", e)
 
         # Shows the list of admins who refused the pending post and removes it form the db
-        show_admins_votes(chat_id=info['chat_id'], message_id=info['message_id'], bot=info['bot'], approve=False)
+        show_admins_votes(
+            chat_id=info['chat_id'], message_id=info['message_id'], bot=info['bot'], approve=False)
         MemeData.remove_pending_meme(info['message_id'], info['chat_id'])
         return None, None, None
```

Comment:
The problem is that right now the caller expects the function to return a tuple with 3 elements to unpack in 3 variables.
```python
return None
```
would cause a **TypeError: cannot unpack non-iterable NoneType object**.

Reply:
ok :+1: